src/test/scala/org/clulab/reach/TestProteinResolutions.scala

```diff
@@ -59,6 +59,15 @@ class TestProteinResolutions extends FlatSpec with Matchers {
     (imkbP.resolve("stat1").isDefined) should be (true)
     (imkbP.resolve("STBA").isDefined) should be (true)
     (imkbP.resolve("stba").isDefined) should be (true)
+    (imkbP.resolve("SMAD2").isDefined) should be (true)
```

Comment:
`imkbP.resolve("SMAD2") shouldBe defined` would provide a more informative error message.
Currently we would get something like "false is not true"